pkg/cvo/status.go

```diff
@@ -377,7 +382,7 @@ func (optr *Operator) syncFailingStatus(original *configv1.ClusterVersion, ierr
 		LastTransitionTime: now,
 	})
 
-	mergeOperatorHistory(config, optr.currentVersion(), now, false)
+	mergeOperatorHistory(config, optr.currentVersion(), false, now, false)
```

Comment:
why set image no verified when failing ?

Reply:
So verified is "level" which false won't reset verified.  We just don't know here, so we don't change the state of the previous history.